src/services/info/notice/notice-detail/NoticeDetailPage.vue

```diff
@@ -62,6 +64,20 @@
             </p-button>
         </section>
         <!--song-lang-->
+        <p-button-modal :header-title="$t('Send the mail')"
+                        :visible.sync="sendEmailModalVisible"
+                        :size="'sm'"
```

Comment:
```suggestion
                        size="sm"
```
그냥 텍스트라면 이렇게 써도 됩니당!